tests/test_autotuner.py

```diff
@@ -123,9 +123,29 @@ def test_mixed_blocking_nthreads():
     assert 'nthreads' in op._state['autotuning'][0]['tuned']
 
 
-def test_tti_aggressive():
+@pytest.mark.parametrize('openmp, expected', [
+    (False, 2), (True, 3)
+])
+def test_mixed_blocking_w_skewing(openmp, expected):
+    grid = Grid(shape=(96, 96, 96))
+    f = TimeFunction(name='f', grid=grid)
+
+    op = Operator(Eq(f.forward, f + 1.), opt=('blocking', 'skewing',
```

Comment:
I'd imagine skewing gets applied before blocking? OK, both ways could work, but logically I think of skewing as a preliminary transformation?

Reply:
hmm.. answered above